web_disable_export_group/templates/assets.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Onestein
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).-->
+<odoo>
+    <template id="assets_backend" name="web_disable_export_group assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/web_disable_export_group/static/src/js/disable_export_group.js"></script>
```

Comment:
End the tag as ```<script.../>```?

Reply:
Old habits die hard. Is it really important? To me it isn't, because it doesn't improve the maintainability nor the functionality.

Reply:
Of course it isn't. It's just more readable and a better habit. It's up to you to change it.